conftest.py

```diff
@@ -26,6 +28,10 @@ def pytest_addoption(parser):
                      type=str,
                      help='Set GCP project to test. Required for GCP tests.')
 
+    parser.addoption('--organization',
```

Comment:
Might be nice to add a `heroku` prefix to this? Something like `--heroku-org`?

Reply:
My thinking was that GitHub also has organizations. Since we only run one service at a time, there wouldn't be confusion.

I'm indifferent, but lean the way it is. How strongly do you feel?

Reply:
Hm, that interesting. I'm unsure, as what you are saying makes perfect sense but it's also different from the current pattern of per-service CLI flags. @g-k what do you think?

Reply:
`--organization` is too ambiguous (AWS has an idea of them too https://aws.amazon.com/organizations/) and we might check multiple services at once (e.g. to audit a shady user's access in aws, heroku, and github), so I'd say `--heroku-org` or `--heroku-organization` (if we're using org or organization we should stick to that).